plugins/xpressive/expressive_plugin.cpp

```diff
@@ -0,0 +1,926 @@
+/*
+ * expressive_plugin.cpp - instrument which uses a mathematical formula parser
+ *
+ * Copyright (c) 2016-2017 Orr Dvori
+ * 
+ * This file is part of LMMS - http://lmms.io
```

Comment:
The repository is full with this mistaken address...
It can be done once with find/replace.